src/sections/Home/So-Special-Section/so-special-style.js

```diff
@@ -39,6 +61,29 @@ const SoSpecialWrapper = styled.div`
             height:100%;
         }    
     }
+    .slick-prev { 
+        left: -2.5rem;
+        position: relative;
+        top: 14.90rem;
+    }
+    .slick-prev:before{	
+        content: "‹";	
+    }
+    .slick-next:before{	
+        content: "›";
+        line-height: 0;
+        opacity: 1;
+        margin-left: 2rem;
```

Comment:
Remove the margin from here, and set `.slick-next` as `right: -2.5rem` at `line: 25` such that both the arrows are equidistant.